english/eating-walls/find-a-toilet/door/alien/use-torch.md

```diff
@@ -4,4 +4,14 @@ The three giant pupils staring at you shrivel into pinholes as the alien backs a
 
 The alien raises his arm, speaks into a device on his wrist, and disappears in a swirl of light.
 
-"Really? A torch!? Who carries a torch around nowadays? What is this, 1992?"
\ No newline at end of file
+"Really? A torch!? Who carries a torch around nowadays? What is this, 1992?"
+
+An echo-ess voice says, "Be it 1992 then..." 
+
+You're teleported to Atlanta-Fulton County Stadium in Atlanta, Georgia on Saturday, October 24, 1992.
+
+Bottom of the 11th 2 outs, Toronto is winning 2 to 1.  The alien appears and offers you a choice to have a ticket with a placed bet for Toronto to win or Atlanta to win.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.